src/images/close.svg

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't know if gatsby already does this, but it might make sense to pass this file throuhg https://jakearchibald.github.io/svgomg/ to clean up unnecessary trash?

Reply:
I have SVGO on my machine and ran it again :)